.github/workflows/python-ci.yml

```diff
@@ -0,0 +1,30 @@
+name: Python CI
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
```

Comment:
It's failing on the Tensorflow installation due to the version of python, it should be `3.7` rather than `3.9`